src/plugins/workspace/public/components/workspace_selector/workspace_selector.tsx

```diff
@@ -0,0 +1,191 @@
+
+const createWorkspaceButton = i18n.translate('workspace.menu.button.createWorkspace', {
+  defaultMessage: 'Create workspace',
+});
+
+const manageWorkspacesButton = i18n.translate('workspace.menu.button.manageWorkspaces', {
+  defaultMessage: 'Manage',
+});
+
+const getValidWorkspaceColor = (color?: string) =>
+  validateWorkspaceColor(color) ? color : undefined;
+
+interface Props {
+  coreStart: CoreStart;
+  registeredUseCases$: BehaviorSubject<WorkspaceUseCase[]>;
+}
+
+export const WorkspaceSelector = ({ coreStart, registeredUseCases$ }: Props) => {
+  const [isPopoverOpen, setPopover] = useState(false);
+  const currentWorkspace = useObservable(coreStart.workspaces.currentWorkspace$, null);
+  const availableUseCases = useObservable(registeredUseCases$, []);
+  const isDashboardAdmin = coreStart.application.capabilities?.dashboards?.isDashboardAdmin;
+
+  const getUseCase = (workspace: WorkspaceObject) => {
+    if (!workspace.features) {
+      return;
+    }
+    const useCaseId = getFirstUseCaseOfFeatureConfigs(workspace.features);
+    return availableUseCases.find((useCase) => useCase.id === useCaseId);
+  };
+
+  const onButtonClick = () => {
+    setPopover(!isPopoverOpen);
+  };
+
+  const closePopover = () => {
+    setPopover(false);
+  };
+  const button = currentWorkspace ? (
+    <EuiPanel paddingSize="none" color="transparent" hasBorder={false} hasShadow={false}>
+      <EuiText
+        size="xs"
+        // here I try inline style to achive the label looks
+        style={{
+          position: 'relative',
+          bottom: '-10px',
+          zIndex: 1,
+          padding: '0 5px',
+        }}
+      >
+        <p>worksapce</p>
+      </EuiText>
+      <EuiPanel paddingSize="s" borderRadius="m">
+        <EuiFlexGroup gutterSize="none" justifyContent="spaceBetween">
+          <EuiFlexItem>
+            <EuiFlexGroup gutterSize="s" justifyContent="flexStart">
+              <EuiFlexItem grow={false}>
+                <EuiIcon
+                  size="l"
+                  type={getUseCase(currentWorkspace)?.icon || 'wsSelector'}
+                  color={getValidWorkspaceColor(currentWorkspace.color)}
+                />
+              </EuiFlexItem>
+              <EuiFlexItem grow={false}>
+                <EuiFlexGroup direction="column" gutterSize="none">
+                  <EuiFlexItem grow={false} style={{ maxWidth: '130px' }}>
+                    <EuiText size="s">
+                      <h4 className="eui-textTruncate">{currentWorkspace.name}</h4>
+                    </EuiText>
+                  </EuiFlexItem>
+                  <EuiFlexItem>
+                    <EuiText size="xs" color="subdued">
+                      <p>
+                        {i18n.translate('workspace.left.nav.selector.description', {
+                          defaultMessage: getUseCase(currentWorkspace)?.title || '',
+                        })}
+                      </p>
```

Comment:
This is `small` as well
```suggestion
                      <small>
                        {i18n.translate('workspace.left.nav.selector.description', {
                          defaultMessage: getUseCase(currentWorkspace)?.title || '',
                        })}
                      </small>
```